rust/Dockerfile

```diff
@@ -0,0 +1,92 @@
+ARG RUST_VERSION=1.58
```

Comment:
Wouldn't it be better not to have a default value?

Reply:
On a second thought, it would, removed.